### PR TITLE
ABS-867 - Update EnabledModuleTabs with Tidbit modules

### PR DESCRIPTION
**Pull request by izhutaev-sugarcrm**
ABS-867 - Update EnabledModuleTabs with Tidbit modules

Update enabled tabs setting for Sugar to include generated by Tidbit modules
Please note that for 7.x QRR need to be performed in order to refresh modules tabs list in UI

### CODE REVIEW

File: src/Helper/ModuleTabs.php [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+    /**
+     * Update enabled modules in top menu.
+     * Add modules that were generated by Tidbit to enabled list
+     *
+     * @param \DBManager $db
+     * @param array $modules
+     */
+    public static function updateEnabledTabs(\DBManager $db, $modules)
+    {
+        global $moduleList;
```
Comment (asorokin-sugar) [inline src/Helper/ModuleTabs.php]: may be we should avoid of using global keyword in new code?

Comment (izhutaev-sugarcrm) [inline src/Helper/ModuleTabs.php]: This is Sugar variable that contains list of modules, that could be displayed in UI and it's instance specific

File: src/Helper/ModuleTabs.php [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+        $sql = "SELECT * FROM config WHERE category = 'MySettings' and name = 'tab'";
+        $result = $db->fetchOne($sql);
+
+        $moduleTabs = ($result)
+            ? unserialize(base64_decode($result['value']))
+            : array();
+
```
Comment (asorokin-sugar) [inline src/Helper/ModuleTabs.php]: check what $moduleTabs is array ?

File: src/Helper/ModuleTabs.php (hunk 1 above)
Comment (asorokin-sugar) [inline src/Helper/ModuleTabs.php]: True, but we should try to prepare external data for process by Tidbit in one place (install_cli.php)
What if Core team make great refactoring in Sugar and remove global?

File: src/Helper/ModuleTabs.php (hunk 2 above)
Comment (izhutaev-sugarcrm) [inline src/Helper/ModuleTabs.php]: basically, if it's not an array - Sugar wont work :) we expect serialized, base64 encoded array type value here

File: src/Helper/ModuleTabs.php (hunk 1 above)
Comment (izhutaev-sugarcrm) [inline src/Helper/ModuleTabs.php]: Ok, good point, I will pass this variable through method params

**Comment by izhutaev-sugarcrm** [comment]
@asorokin-sugar I've updated this PR

**Pull request merged**